access/resource_secret_acl_test.go

```diff
@@ -142,6 +252,27 @@ func TestResourceSecretACLRead_Error(t *testing.T) {
 	assert.Equal(t, "global|||something", d.Id(), "Id should not be empty for error reads")
 }
 
+func TestResourceSecretACLRead_ScopeWithSlash_Error(t *testing.T) {
```

Comment:
This test is not necessary as well - url encoding of a slash was tested by the first test added